libraries/AP_Airspeed/AP_Airspeed.cpp

```diff
@@ -114,7 +114,8 @@ const AP_Param::GroupInfo AP_Airspeed::var_info[] = {
 #ifndef HAL_BUILD_AP_PERIPH
     // @Param: _USE
     // @DisplayName: Airspeed use
-    // @Description: Enables airspeed use for automatic throttle modes and replaces control from THR_TRIM. Continues to display and log airspeed if set to 0. Uses airspeed for control if set to 1. Only uses airspeed when throttle = 0 if set to 2 (useful for gliders with airspeed sensors behind propellers).
+    // @Description{Plane}: Enables airspeed use for automatic throttle modes and replaces control from THR_TRIM. Continues to display and log airspeed if set to 0. Uses airspeed for control if set to 1. Only uses airspeed when throttle = 0 if set to 2 (useful for gliders with airspeed sensors behind propellers).
+    // @Description{Copter, Blimp, Rover}: This parameter is not used by this vehicle. Always set to 0.
```

Comment:
Actually in Sailboats they are used....so this is not entirely correct....rather than trying to explain, lets just leave it that the PARAMETER is not used and should be set to 0. The new wiki pages has the information...